Added ⚡️ content to `Onboarding` > `Backups` (#541)
BitcoinDesign/Guide: guide/onboarding/backing-up-a-wallet.md

image: https://bitcoin.design/assets/images/guide/onboarding/backing-up-a-recovery-phrase/backing-up-a-recovery-phrase-preview.png
---

<!--

Editor's notes

This chapter covers backing up a recovery phrase during user onboarding.

Illustration sources

- https://www.figma.com/file/qr4P17z6WSPADm6oW0cKw2/?node-id=0%3A1
- https://www.figma.com/file/q9EgLqOKcIVc0Cq7khtpNm/Onboarding-%3E-Backups?node-id=0%3A1

-->

{% include picture.html
   image = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/backing-up-a-recovery-phrase.png"
   retina = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/backing-up-a-recovery-phrase@2x.png"
   mobile = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/backing-up-a-recovery-phrase-mobile.png"
   mobileRetina = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/backing-up-a-recovery-phrase-mobile@2x.png"
   alt-text = "Backing up a recovery phrase chapter header image"
   width = 1600
   height = 600
   layout = "full-width"
%}

# Backing up a wallet

When a user generates a new Bitcoin wallet, they will soon need to create a backup of their wallet in order to ensure that funds are not lost.

> There are two main types of information that need to be backed up: the user's wallet recovery data and their Lightning channel state. While the wallet recovery data may be used to restore the wallet using another wallet application, the channel state is typically only restorable within the same application.

Reviewer: Slight simplification.
```suggestion
Two main types of information need to be backed up: the user's wallet recovery data and their Lightning channel state. While the wallet recovery data may be used to restore the wallet using another wallet application, the channel state is typically only restorable within the same application.
```

{% include picture.html
image = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/wallet-backup.png"
retina = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/wallet-backup@2x.png"
mobile = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/wallet-backup-mobile.png"
mobileRetina = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/wallet-backup-mobile@2x.png"
alt-text = "Example image"
width = 1200
height = 680
layout = "full-width"
%}

> Users who lose both their wallet access and their backup permanently lose their bitcoin. Arguably, this creates one of the toughest design problems within the space.

Reviewer: ```suggestion
Users who lose both their wallet access and their backup permanently lose their bitcoin. Arguably, this creates one of the most challenging design problems within the space.
```

> Backups could be handled automatically by the software, manually by the user, or a mix of these two methods. Regardless of the type of scheme you decide to go with, it’s crucial to be explicit about how your product is handling backups and private key management.

Reviewer: ```suggestion
Backups could be handled automatically by the software, manually by the user, or a mix of these two methods. Regardless of the type of scheme you decide to go with, it’s crucial to be explicit about how your product is handles backups and private key management.
```

Reviewer 2: Need to remove is

{% include tip/recommendation.html %}

> We recommend that the optimal phase to hint the user to perform the backup should be after the wallet has received funds for the first time. This way, we avoid overwhelming the user with an unnecessary task, as it makes more sense to backup a wallet with funds on it.

Reviewer: @sbddesign Do you have any ideas on how we may want to communicate this differently with regards to the [discussion](https://bitcoindesign.slack.com/archives/C015856BDME/p1633778519103100) on Slack:
![Screenshot 2021-11-02 at 11 12 13](https://user-images.githubusercontent.com/36959754/139827720-7ecd33de-0b66-4d9d-adcd-5e3241819ded.png)

Reviewer 2: I will add some details around this in the `Funding a wallet` page too.

Author: @pavlenex Are you sayin we should not suggest waiting until first receipt for backup, or are you saying we should mention that it depends on how much bitcoin is at stake, or otherwise talk about the pros/cons of this approach?

Reviewer 3: A nice approach could be to ask users to do the backup during onboarding, with an option to "Do this later". If they choose to do it later, the app can place a small reminder on the home screen, and possibly reinforce this with a larger info panel once the first bitcoin have been received.

This way, users were made aware of the backup so they know it's an important thing to do. And people who want to skip it (just want to try the app, are in a public space, etc) can move past quickly and don't get stuck.

Reviewer: I think we can merge this one, since this change is not related to this PR and discuss the approach(es) in a new issue if this is something we'd like to pursue.

Reviewer 3: I concur.

{% include tip/close.html %}

Recent [studies](https://dl.acm.org/doi/fullHtml/10.1145/3411764.3445679) show users are often confused or unaware of where their private keys are being stored, ultimately causing inadequate risk assessment and poor storage behavior. This risk of user negligence may be avoided with transparent messaging around the scheme your wallet is using.

> New-to-crypto users often expect a recovery mechanism that allows them to “regain access to their funds in the case that they lose their seed phrase.”
>
> <cite>As researched by <a href="https://dl.acm.org/doi/fullHtml/10.1145/3411764.3445679">Voskobojnikov et al</a></cite>

> Cloud backups may be preferable in many situations. They speed up the onboarding process, are easier for newer users, and protect the user from mishandling their recovery phrase and potentially losing funds. They can also be automated to help ensure that there is an up-to-date backup of channel state in case the user loses their wallet. This type of gateway gives a beginner security without overwhelming them with unfamiliar onboarding actions.

Reviewer: Just a suggestion on condensing:

```suggestion
Cloud backups may be preferable in many situations. They speed up the onboarding process, are easier for newer users, and protect the user from mishandling their recovery phrase and potentially losing funds. They can also be automated to help ensure an up-to-date backup of channel state if the user loses their wallet. This type of gateway gives a beginner security without overwhelming them with unfamiliar onboarding actions.
```
